pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -106,6 +108,15 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 				Name:    pf.Name(),
 			}
 		} else {
+			if completedVersion := minorUpdateFrom(cv.Status, currentVersion); completedVersion != "" && patchOnly {
+				// This is to generate an accepted risk for the accepting case 4.y.z -> 4.y+1.z' -> 4.y+1.z''
+				return &precondition.Error{
+					Reason:             "MinorVersionClusterUpgradeInProgress",
+					Message:            fmt.Sprintf("Retarget to %s while a minor level upgrade from %s to %s is in progress", targetVersion, completedVersion, currentVersion),
```

Comment:
micro-nit, `MinorVersionClusterUpgradeInProgress`  and `a minor level upgrade from` both include "upgrade", where we prefer "update" (openshift/openshift-docs#40525).  Some things like `Upgradeable` predate that attempt at consistency, but for these net-new strings, I think we should align with the current docs conventions.

Reply:
Updated with "update". 😄